src/types.ts

```diff
@@ -112,3 +112,5 @@ export interface TriggerQueueUpdatePayload {
   id: number
   status: TriggerStatus
 }
+
+export type ApplicationOutcome = 'Pending' | 'Approved' | 'Rejected'
```

Comment:
Another case for using the exported schema with types?

Reply:
I think I'll have to get you to show me how to do this. The auto-generated `schema.graphql` file is not in a format that can be imported into Typescript file. How did you generate the `graphql.tsx` file in the front-end? 🤷‍♂️

Reply:
Cool -- I've updated to auto-generate and use types from Schema. Cheers. 👍

Reply:
Also updated the `database_init` script to auto-generate the types file.

Reply:
Nice!

Reply:
I was thinking it would be good to have this generator automated somewhere in the front-end as well. Not sure maybe the pre-commit hook